R/tuning.R

```diff
@@ -74,6 +74,44 @@ tuning <-
   }
```

Comment:
We need to handle the possible exception that the tuning function returned multiple lambda's. I think a good strategy would be to first check the length of lambda, if there are indeed multiple lambdas, raise a warning then return the smallest, that is:
```
lambda_selected <- do.call(func_name, list(Y = Y, K_mat = K_mat, lambda = lambda))
if (length(lambda_selected) != 1){
    warning(paste0('Multiple (', length(lambda_selected), ') optimal lambda's found, returning the smallest.')
}
min(lambda_selected)
```